sections/other_rules.tex

```diff
@@ -46,14 +46,21 @@ \subsection*{\hypertarget{Difficulty}{Difficulty}}
 
 All \textbf{Artifacts} received from a starting bonus should be placed into your \textbf{hand} and not shuffled into your Starting Deck.
 If you searched for any Artifacts, shuffle the Artifact Deck and its Discard Pile\index{Discard Pile} together afterwards, and then discard one Artifact from the top to form the Artifact Discard Pile again.\par
-The chosen difficulty also determines the number and type of neutral enemies that are encountered during Neutral Combat according to the table below.
+The chosen difficulty also determines the number and type of neutral enemies that are encountered during Neutral Combat according to the \hyperlink{Difficulty Table}{table} at the back cover of the book.
+
+\end{multicols}
 
 \vfill
-\hfill\includegraphics[width=0.8\linewidth]{\art/cerberi.png}
+\includegraphics[width=\linewidth]{\art/sharpshooter.jpg}
 
-\end{multicols}
+\clearpage
 
 \begin{figure*}[!hb]
   \centering
-  \includegraphics[width=\linewidth]{\tables/field_difficulty.png}
+  \includegraphics[width=\linewidth]{\tables/opt_rules.png}
 \end{figure*}
+
+\begin{scaledfigure}[blanker]
+  \centering
+  \includegraphics[width=\linewidth, height=\myspace, keepaspectratio]{\art/magic_arrow.png}
```

Comment:
Magic Arrow doesn't fit here. Can you try something else? Actually, a full framed graphic would fit better, not a transparent one. If you find an appropriate graphic in any of the original rule books, I can rip it for you. It must be wide but not tall.

Reply:
Couldn't find anything small enough. Will just remove it